expression_data_matrix.md

```diff
@@ -24,6 +24,55 @@ The following metadata fields describe a minimal set of metadata necessary to wo
 
```

Comment:
Having an actual schematic view of all the matrices will be useful. 
Sean, I gave you the edit access on the google doc I used a while back for a presentation during our bi-weekly meeting. Feel free to modify/use that drawing here:
`_Insert image of the expression matrix model_`